src/assets/stylesheets/screens/_sign-in.scss

```diff
@@ -0,0 +1,26 @@
+.container-sign-in {
+  display: flex;
+  flex-direction: column;
+  width: rem(320px);
+}
+
+.form-input-group {
+  display: flex;
+  flex-direction: column;
+  width: rem(320px);
+  height: rem(100px);
+
+  &__label {
+    margin-bottom: rem(8px);
+
+    color: map-get($brand-white, 'primary');
+    font-size: rem(15px);
```

Comment:
@gutakk Should we set the `font-weight` here?
![Screen Shot 2564-06-18 at 19 19 31](https://user-images.githubusercontent.com/1772999/122560011-3fc64000-d06a-11eb-836c-d25951d59aba.png)

Reply:
Not sure how I missed that 😢. Fixed in 2c00d7895e87753d219aa1931c3b992f046c3627